install.sh

```diff
@@ -0,0 +1,14 @@
+#! /usr/bin/env bash
+
+if [[ $ASSETS == 'nobuild' ]]; then
+    composer install --no-interaction --no-progress --prefer-dist -o
+else
+    echo "Installing PHP dependencies through Composer..."
+    composer install
```

Comment:
So the composer install command should be `SYMFONY_ENV=prod composer install --no-dev -o --prefer-dist`.
But this shouldn't be that command for test & dev :confused: 

Should we have a install_dev.sh ?